ocfweb/docs/docs/services/hpc.md

```diff
@@ -0,0 +1,160 @@
+well as the amount of computational resources that are expected to be needed.
+
+### Connecting
+
+Once you submit your proposal and are approved access you will be able to
+connect to our Slurm master node via SSH by running the following command:
+
+```
+ssh my_ocf_username@hpcctl.ocf.berkeley.edu
+```
+
+If you have trouble connecting please [[contact us|doc contact]] at
+[help@ocf.berkeley.edu](mailto:help@ocf.berkeley.edu), or come to
+[[staff hours|staff-hours]] when the lab is open and chat with us in person.
+We also have a #hpc_users channel on [slack][fco] and [[irc|doc contact/irc]]
+where you can ask questions and talk to us about anything HPC.
+
+## The Cluster
+
+Users are currenly able to run jobs on our main HPC server, which has 4
+1080Ti's and 256GB of DDR4-2400 RAM, which was funded by generous grants from
+the Student Tech Fund
+
+## Slurm
+
+We currently use [Slurm][slurm] as our workload manager for the cluster. We
+will soon post technical documentation about our Slurm configuration, but briefly,
+Slurm is a free and open-source job scheduler which helps distribute jobs from
+all users evenly among HPC computers, referred to as nodes. In short, all of
+your programs will be run through Slurm. To use Slurm there are several commands
+that will be helpful:
+
+* `srun`: Used to submit jobs.
+* `scontrol`: Used to view and edit the Slurm configuration.
+* `squeue`: Used to view running and queued jobs.
+
+## Dependencies
+For managing application dependencies, you currently have two options:
+
+### Virtual Environments
+
+First you can use a [virtual environment][venv] if you are using Python
+packages. To create a virtual environment navigate to your home directory
+and run the following commands:
+
+```
+virtualenv -p python3 venv
+. venv/bin/activate
+```
+
+This will allow you to *pip install* any Python packages that the OCF does not
+already have for your program.
+
+### Singularity
+
+For those who need access to non-Python dependencies or have already integrated
+their program into Docker, the second option is to use Singularity containers.
+[Singularity][singularity] is a containerization platform developed at Lawrence
+Berkeley National Laboratory that is designed specifically for HPC environments.
+To read more about the benefits of Singularity you can look
+[here][singularity_article]. We suggest a particular workflow, which will help
+simplify deploying your program on our infrastructure.
+
+#### Installing
+
+We recommend that you do your development on our HPC infrastructure, but you
+can also develop on your own machine if you would like. If you are running
+Linux on your system, you can install Singularity from the official apt repos:
+
+```
+sudo apt install singularity-container
+```
+
+If you are running Mac you can look [here][mac_install], or Windows [here][win_install].
+
+#### Building Your Container
+
+```
+singularity build --sandbox ./my_container docker://ubuntu
```

Comment:
Since `--sandbox` is an optional flag, maybe wrap it in square brackets to denote that it's optional?